tools/samtools_fastq.cwl

```diff
@@ -44,7 +44,7 @@ inputs:
   input_reads_1: {type: File, doc: "For FASTQ input, please enter reads 1 here. For BAM input, please enter reads here."}
   input_reads_2: {type: "File?", doc: "For FASTQ input, please enter reads 2 here. For BAM input, leave empty."}
   SampleID: string
-  runThreadN: int 
+  runThreadN: { type: 'int?', default: 16 } 
```

Comment:
Should the default threads be 36 since that's what the previous version was?

Reply:
Hmmm, it was? I might have to ensure in the new wf, and refactored wf that it gets it's own cores param. Currently at WF level, it just get's passed to everything